content/posts/book/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Newly released open access book"
+date: 2021-11-15T14:26:51+01:00
+draft: false
+description: "New open access book released"
+categories: ["News"]
+displayInList: true
+author: Nicolas P. Rougier
+resources:
+- name: featuredImage
+  src: "book-cover.png"
+  params:
+    description: "Book cover"
+    showOnTop: true
+---
+
+It's my great pleasure to announce that I've finished my book on matplotlib and it is now freely available at https://www.labri.fr/perso/nrougier/scientific-visualization.html while sources for the book are hosted at github.com/rougier/scientific-visualization-book.

```

Comment:
Can you add a hyperlink for these two URLs? See here on how https://commonmark.org/help/tutorial/07-links.html

Reply:
Fixed